samples/sensor/hx711/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _hx711:
+
+HX711: 24-Bit Analog-to-Digital Converter (ADC) for Weigh Scales
```

Comment:
The commit message is about adding a driver, but this is adding an application. Please update.

Reply:
done.